### DISCUSSION TITLE
Does reStructuredText support `--citeproc`?

### DISCUSSION THREAD

**Discussion by marshallward** [jgm/pandoc#9901] [General]
I've been trying out the `--citeproc` flag with a reStructuredText file, but it seems to ignore the `[@..]` syntax.  Most of the examples also appear to be in Markdown (which seems to work for me).  Is this feature specific to Markdown?  Or simply ignored in reStructuredText?  Or am I just doing it wrong?  Any advice would be great, thanks!

**Comment by jgm** [comment]
`--citeproc` itself will work with any input format that generates `Cite` elements in the AST.
Currently that's markdown, typst, org, latex, and docx. As far as I know, RST doesn't have a special syntax for citations that should parse to `Cite`.

**Comment by marshallward** [comment]
I see, thanks.  At first I was hoping there was some way to integrate [reST citations](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#citations), but I suppose they're not quite the same thing.

There is the [sphinx bibtex extension](https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html#usage), which AFAIK is fairly widely used, but perhaps that deviates too far from standard reST.

Anyway, thanks for the update, I'll find some way to do it outside of pandoc.

**Comment by marshallward** [comment]
Actually if you don't mind, I'd like to try and probe this a bit more.

I presume the problem is that reST lacks some kind of formal means of citation, such as in docx and org.  Most of these markups appear to be aiming for LaTeX and BibTeX integration.

If I look to Markdown, there is no common standard (or perhaps I should say there are many standards), but there appears to be some kind of consensus around the `@` symbol for LaTeX-like citation.

For better or worse, reST is very unlikely to change much in the future.  But I believe that sphinxcontrib-bibtex has gotten most of the details right.  At the very least, the `:cite:` and `.. bibliography` directives, combined with a CSL file, would get one very close to a LaTeX-like solution.

So getting to my question: Is there any possibility that pandoc would consider support for `:cite:` and `.. bibliography`?  Normally I would never suggest that pandoc deviate from reST.  As for myself, I try very hard to not rely on Sphinx-specific directives.  But in this case, everything points in the right direction.  And the alternative would simply be that reST never supports citations.

I don't want to suggest work for anyone, but if I were to at least try to fork pandoc and explore this, is there any advice on where to start?

**Reply by jgm** [reply]
Yes, we could consider supporting the Sphinx citations in RST (when the `citations` extension is set).  Submit an issue for this?

**Reply by marshallward** [reply]
Sure, thanks for considering it.  I will put in an issue.
